osmcha/changeset.py

```diff
@@ -285,6 +287,35 @@ def is_whitelisted_editor(self):
                 is_whitelisted = True
         return is_whitelisted
 
+    def autovandal(self):
+
+        reason = 'Flagged by ML classifier'
+
+        def load_model():
+            return joblib.load('models/autovandal.pkl')
+
+        def changeset_to_data(changeset):
+            return [
+                len(changeset.editor),
```

Comment:
I know. We need to add feature hashing to autovandal. Per voice, we are skipping this string columns for now.